README.md

```diff
@@ -12,10 +12,37 @@ documentation for [Python][1] or [Java][2].
 
 ## Install
 
-```bash
-go get -u github.com/GoogleCloudPlatform/go-endpoints/endpoints
+Start with `go get github.com/GoogleCloudPlatform/go-endpoints/endpoints`. If this is not
+the first time you're "getting" the package, add `-u` param to get an updated
+version, i.e. `go get -u ...`.
+
+Now, you'll see a couple errors:
+
+```
+package appengine: unrecognized import path "appengine"
+package appengine/user: unrecognized import path "appengine/user"
+package appengine_internal/user: unrecognized import path "appengine_internal/user"
 ```
 
+which is OK, don't worry! The issue here is Go looks at all imports in
+`endpoints` package and cannot find "appengine/*" packages nowhere in your
+`$GOPATH`. That's because they're not there, indeed. Appengine packages are
+normally available only when running an app with dev appserver, and since that's
+precisely what we want to do, "unrecognized import path" errors can be safely
+ignored.
+
+
+Alternatively, use [goapp tool][goapp] from the Google App Engine SDK for Go
```

Comment:
I would change the order so the non error version goes first, then basically say: if you don't have goapp installed (which is weird ... because you're using app engine) you can also run `go get` which will show these errors.